.eslintrc.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     'plugin:@yext/export-star/recommended'
   ],
   plugins: ['react-perf'],
-  ignorePatterns: ['lib', 'tests/setup/responses'],
+  ignorePatterns: ['lib', 'tests/setup/responses', 'scripts'],
```

Comment:
oh we can get rid of scripts here for now

Reply:
yea, my b